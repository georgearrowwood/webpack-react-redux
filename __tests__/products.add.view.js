import React from 'react';
import ProductAdd from '../app/components/products/add';
import renderer from 'react-test-renderer';

test('Render product add form', () => {
  const component = renderer.create(
    <ProductAdd/>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
