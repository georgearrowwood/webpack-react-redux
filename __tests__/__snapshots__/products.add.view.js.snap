// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render product add form 1`] = `
<div>
  Add Product:
  <div>
    <input
      onChange={undefined}
      type="text"
      value={undefined}
    />
    <button
      onClick={undefined}
    >
      Add
    </button>
  </div>
</div>
`;
